template/hooks/php/cs-fixer/fix-cs

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -u
+. "$GIT_DIR/hooks/git_config_wrapper.sh"
+
+fix_cs()
+{
+	get_hook_config php-cs-fixer executable executable optional php-cs-fixer
+	$executable fix --verbose
```

Comment:
With PHP-CS-Fixer v2, you can specify file to fix to make it quicker: https://github.com/FriendsOfPHP/PHP-CS-Fixer/tree/v2.0.0-alpha#using-php-cs-fixer-on-ci